Automattic/jetpack
# Brute Force Protection: Use "allow list" instead of "whitelist"

## Proposed changes:

* Updates instances of "whitelist" and "white list" to use our standard "allow list" terminology.
* Does not update existing option names, just user messages and internal comments/variable names.

### Other information:


## Jetpack product discussion



p7pQDF-6yH-p2

1201069996155224-as-1203767209050394

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Proof read changes.
* Check for regression - toggle brute force protection, validate the math problem appears, validate the IP allow list works.
